Disaster Recovery/SafeHaven 5 General/resize-linux-pg.md

```diff
@@ -0,0 +1,239 @@
+{{{
+  "title": "Expand Linux Protection Group",
+  "date": "03-16-2020",
+  "author": "Shi Jin",
+  "attachments": [],
+  "contentIsHTML": false
+}}}
+
+### Outline
+
+Suppose a Linux server protected by SafeHaven-5 has been expanded, in other words running on a larger file system,  a number of steps needs to be taken in order to sure the SafeHaven protection continue to work with the enlarged system. At the high leve, the steps are
+
+1. Using the SafeHaven GUI, [expand the SafeHaven protection group and protected disk](../SafeHaven 4/SafeHaven-4-Expand Protection Group Size.md):
+  1.1 Ensure both SRNs contain sufficient free capacity in the correspondig storage pools
+  1.2 Enlarge the protection group
+  1.3 Enlarge the protected disk within the protection group
+
+2. On the protected Linux server itself, run as root user to
+  2.1 ensure the iSCSI Disk has been expanded
+  2.2 expand the root file system partition
+  2.3 expand the root file system
+  2.4 test to confirm
```

Comment:
In 2.1 .. 2.4 would be good to start with a capital letter, for consistency with 1.1 .. 1.3.